rudder-server-root/SPECS/rudder-server-root.spec

```diff
@@ -52,7 +52,7 @@ Source8: rudder-server-root
 BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 BuildArch: noarch
 
-Requires: rudder-jetty, rudder-webapp, rudder-inventory-endpoint, rudder-inventory-ldap, rudder-reports, rudder-agent, curl
+Requires: rudder-jetty, rudder-webapp, rudder-inventory-endpoint, rudder-inventory-ldap, rudder-reports, Rudder::Agent, curl
```

Comment:
The or operator is supported for dependencies as show here : 
https://docs.fedoraproject.org/en-US/Fedora_Draft_Documentation/0.1/html/RPM_Guide/ch-specfile-syntax.html#id729642

So you could use "Rudder::Agent or rudder-agent"